uniflow-core/src/main/kotlin/io/uniflow/core/result/SafeTypeBuilders.kt

```diff
@@ -1,46 +1,46 @@
 package io.uniflow.core.result
 
-import arrow.core.Try
+import arrow.core.Either
 
-suspend fun <T> safeValue(expr: suspend () -> T): Try<T> {
+suspend fun <T> safeValue(expr: suspend () -> T): Either<Throwable, T> {
     return try {
-        Try.Success(expr())
+        Either.right(expr())
     } catch (exception: Throwable) {
-        Try.Failure(exception)
+        Either.left(exception)
     }
 }
 
-suspend fun <T> safeValue(expr: suspend () -> T, onError: (Throwable) -> Throwable): Try<T> {
+suspend fun <T> safeValue(expr: suspend () -> T, onError: (Throwable) -> Throwable): Either<Throwable, T> {
```

Comment:
This can be delegated to `Either#catch()` instead for simplicity. There's an overload that provides a lambda for mapping the error before lifting it to the left side.